celobox/phantompasswd.py

```diff
@@ -76,7 +77,10 @@ def load_data(self, domain):
 
         for content_type in ContentTypes:
             try:
-                content = open(os.path.join(manifests_path, '{}.{}'.format(domain, content_type.value)))
+                content = open(resource_filename(
+                    __name__,
+                    os.path.join(manifests_path, '{}.{}'.format(domain, content_type.value))
+                ))
```

Comment:
Noticed that myself, and I assumed that was your plan, but figured we could address that when we worked on that.

I'll push an update to the PR that moves it there. I think (but I'm not 100% sure) that it is safe to use resource_filename to get a path, which we can os.path.join to. I'll see if I can verify that.

Another common pattern is to use resource_filename during installation to copy the files to some specific path, and then the code can just call them from there. So you would put them somewhere like /usr/local/share/celobox/manifests/ (probably depending on the distro/operating system) and the code would load them up from there.